### PR TITLE
Enterprise release management in parallel

### PR DESCRIPTION
**Pull request by orangejenny**
##### SUMMARY
https://dimagi-dev.atlassian.net/browse/COV-21

Review by commit.

##### RISK ASSESSMENT / QA PLAN
[QA ticket](https://dimagi-dev.atlassian.net/browse/QA-1650)

### CODE REVIEW

**Comment by orangejenny** [comment]
@sravfeyn While learning about chords, I saw [this comment](https://github.com/dimagi/commcare-hq/blob/58cbddea1aa3c6055e4ee1e5c14e9e33c3134737/corehq/messaging/tasks.py#L168-L169). If you have any feedback on this PR, I'm interested.

**Review by calellowitz — APPROVED** [review]

File: corehq/apps/linked_domain/tasks.py [hunk 1]
```diff
@@ -223,3 +206,30 @@ def release_domain(domain_link, username, models, build_apps=False):
             manager._add_success(domain_link.linked_domain, _("Updated {} successfully").format(model['name']))
 
     return manager.results()
+
+
+@task(queue='background_queue')
+def send_email(results, master_domain, username, models, linked_domains):
```
[the remark this answers is not in the document]
Comment (orangejenny) [reply corehq/apps/linked_domain/tasks.py:212]: sure, 61ab05ab39d

[the remark this answers is not in the document]
Comment (orangejenny) [reply corehq/apps/linked_domain/tasks.py]: sure, 61ab05ab39d

**Comment by sravfeyn** [comment]
It looks like Giovanni added that comment [originally](https://github.com/dimagi/commcare-hq/pull/19027/files#diff-954487d92668ca2c42f33fa7c475970eR117), I am actually not familiar with celery chord. Sorry, @orangejenny

**Comment by orangejenny** [comment]
Thanks @sravfeyn , no worries, we'll see how this performs ;)

**Pull request merged**